File: src/main/resources/export/java/TimedCommand-constructor.java
#set($command = $helper.getByName($command_name, $robot))

## parameters are hardcoded -- Timed Commands only take a single double argument

    public #class($command.name)() {
        this(0);
    }

    public #class($command.name)(double timeout) {
        super(timeout);
#if ( $command.getProperty("Run When Disabled").getValue() )
        setRunWhenDisabled(true);
#end

File: src/main/resources/export/cpp/TimedCommand-constructor.cpp
#set($command = $helper.getByName($command_name, $robot))

#class($command.name)::#class($command.name)(double timeout) : frc::TimedCommand(timeout) {
    // Use AddRequirements() here to declare subsystem dependencies
    // eg. AddRequirements(Robot::chassis.get());
    SetName("#class($command.name)");
    #if  (${command.getProperty("Requires").getValue()} != "None")
    AddRequirements(#variable(${command.getProperty("Requires").getValue().toLowerCase()}));
    #end
